src/streamer.cpp

```diff
@@ -528,6 +528,7 @@ void Streamer::executeCallbacks()
 				int amxIndex = 0;
 				if (!amx_FindPublic(*i, "Streamer_OnItemStreamIn", &amxIndex))
 				{
+                    amx_Push(*i, static_cast<cell>(c->get<2>()));
```

Comment:
Could you fix the indentation here (also on line 601)?

Reply:
Done